indico/modules/events/controllers/display.py

```diff
@@ -28,16 +28,29 @@ class RHExportEventICAL(RHDisplayEventBase):
     @use_kwargs({
         'scope': fields.Enum(CalendarScope, load_default=None),
         'detail': fields.String(load_default=None),
-        'series': fields.Boolean(load_default=False)  # Export the full event series
+        'series': fields.Boolean(load_default=False),  # Export the full event series
+        'reminder': fields.Int(load_default=None)
```

Comment:
```suggestion
        'reminder': fields.Int(load_default=None, validate=validate.Range(0)),
```

That way you disallow negative values.